rebar.config

```diff
@@ -17,17 +17,9 @@
 
 {deps, [{katana_code, "1.1.2"}]}.
 
-{project_plugins,
- [{rebar3_hex, "~> 6.11.6"}, {rebar3_lint, "~> 0.4.1"}, {rebar3_hank, "~> 1.1.3"}]}.
-
-{dialyzer,
- [{warnings, [no_return, unmatched_returns, error_handling, underspecs]},
-  {get_warnings, true},
-  {plt_apps, top_level_deps},
-  {plt_extra_apps, []},
-  {plt_location, local},
-  {base_plt_apps, [erts, kernel, stdlib, runtime_tools, syntax_tools]},
-  {base_plt_location, "_build/global_plt"}]}.
+{project_plugins, [rebar3_hex, rebar3_lint, rebar3_hank]}.
```

Comment:
I bumped all these to latest to (hopefully) ease maintenance; hope that's Ok.

Reply:
Nope. That's against NextRoll's policy. We like our builds to be _predictable_ and we use https://github.com/AdRoll/rebar3_depup to stay up-to-date, instead.

Reply:
Cool. Reverting.

Reply:
I already reverted… oops! I did it incorrectly 🤦🏻

Reply:
Now fixed for good.

Reply:
150a4a9

Reply:
It seems we were changing it at the same time 😄